e2e/core/viewer/viewer-component.e2e.ts

```diff
@@ -203,7 +203,7 @@ xdescribe('Viewer', () => {
 
     });
 
-    describe('PowerPoint Folder Uploaded', () => {
+    xdescribe('PowerPoint Folder Uploaded', () => {
```

Comment:
please do not exclude any e2e

Reply:
These have been excluded, because there are related bugs - ADF-3953 and ADF-3963

Reply:
In general, we don't comment out a test. The options are:

1. There is a regression we fix the regression
2. The use case is not valid anymore we delete the test
3. The test has a problem we fix the test

Reply:
I think, the use case is valid, the Test seems to be fine as well. the 2 Jira's need fixing ADF-3953 and ADF-3963